### PR TITLE
feat: created ONBOARDING.md doc.

### PR DESCRIPTION
**Pull request by FelipeCoro**
## Summary
- Created `ONBOARDING.md` doc with instructions for local kit and core integration testing in the sample app.

## Testing Plan
- All test passed.

## Reference Issue
- Closes https://go.mparticle.com/work/SQDSDKS-4468

### CODE REVIEW

File: core-sdk-samples/higgs-shop-sample-app/ONBOARDING.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#### A good way to test kit integration is by placing a breakpoint inside the kit's onCreate() and run the app on debug mode, the breakpoint should trigger indicating the kit is being initialized.
+
+
+# Adding the SDK to a local mParticle sample app project.
+
+Do to the complexity of the multi-module configuration of the SDK adding it directly as an operable module in the sample app is not currently supported, different steps must be taken to *build the SDK locally and add it as a dependency.*
+
+1. First fork a local copy of the SDK repository, in this local copy you must make *all the configuration changes that you wish to see implemented on your local sample app* before you make a local build.
+
+2. Once your changes are made run the following command  `./gradlew publishReleaseLocal`  this will create a local snapshot of the SDK.
```
Comment (MystiBerry) [inline core-sdk-samples/higgs-shop-sample-app/ONBOARDING.md:32]:
```suggestion
2. After your changes are made, run the command  `./gradlew publishReleaseLocal`.  This command creates a local snapshot of the SDK.
```

File: core-sdk-samples/higgs-shop-sample-app/ONBOARDING.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+3. Finally in the sample app's *module-level build.gradle.kts* file replace the original core implementation with yours, you can find the name in your local **m2** folder it should contain the word "SNAPSHOT" in it, the complete dependency should look like this: `implementation("com.mparticle:android-core:#.##.#-SNAPSHOT"`
```
Comment (MystiBerry) [inline core-sdk-samples/higgs-shop-sample-app/ONBOARDING.md:34]:
```suggestion
3. Finally, in the sample app's *module-level build.gradle.kts* file, replace the original core implementation with yours. You can find the name in your local **m2** folder. The dependency should contain the word "SNAPSHOT." The complete dependency should be similar to the following: `implementation("com.mparticle:android-core:#.##.#-SNAPSHOT"`.
```

File: core-sdk-samples/higgs-shop-sample-app/README.md [hunk 3]
```diff
@@ -101,6 +101,10 @@ To verify that your events have arrived at mParticle's servers, or to compare yo
 
 This will not only show your data as it enters mParticle, but also as your data is forwarded to our various partner services and integrations (if enabled).
 
+### Integrating kits and core SDK locally in the sample app
+
+For information on how to integrate a kit or the core SDK for local configurations please refer to [ONBOARDING.md](./ONBOARDING.md)
```
Comment (MystiBerry) [inline core-sdk-samples/higgs-shop-sample-app/README.md:106]:
```suggestion
For information on how to integrate a kit or the core SDK for local configurations refer to [ONBOARDING.md](./ONBOARDING.md) 
```

**Review by MystiBerry — COMMENTED** [review]
I made some suggestions, mostly for brevity or clarity. Great instructions!

**Comment by FelipeCoro** [comment]
@MystiBerry Great Mysti! thank you for the feedback, a new PR with the changes has been made.

**Comment by mchuangatmp** [comment]
@MystiBerry sorry about force pushing the new changes on same commit, we won't do that in future so it's easier to see the diff next time

**Review by MystiBerry — APPROVED** [review]

**Review by mchuangatmp — APPROVED** [review]

**Pull request merged**